docs/tutorials/clickhouse-metabase.rst

```diff
@@ -0,0 +1,144 @@
+
+After all, a picture is worth a thousand words.
+
+Preparing ClickHouse cluster and data
+--------------------------------------
+
+We'll start on the ClickHouse side. In this section we'll set up the cluster and load the data that we can use for analysis.
+
+Setting up Aiven for ClickHouse
++++++++++++++++++++++++++++++++
+
+We recommend you use `Aiven for ClickHouse <https://aiven.io/clickhouse>`_ when following this tutorial. If you still don't have Aiven account, `register over here <https://console.aiven.io/signup>`_. You'll get free credits that you can use for this tutorial.
+
+Once you're logged into  Aiven platform, click on **Create service** and follow the wizard to set up the preferences.
+
+
+We also have `a documentation <https://docs.aiven.io/docs/products/clickhouse/getting-started>`_  for detailed instructions.
+
+Loading data
+++++++++++++++
+If you already have data in your ClickHouse database for analysis, feel free to skip this step and visualise your data.
+
+Alternatively, you can use `a dataset from New York Public Library "What's on the Menu?" <http://menus.nypl.org/data>`_ as an example. This dataset is easy to setup and it has ample opportunities for exploration. The official documentation of ClickHouse offers `a convenient set of instructions <https://clickhouse.com/docs/en/getting-started/example-datasets/menus/>`_ to create the tables and load the data.
+
+We'll be utilizing the data from the "What's on the Menu?" dataset in the examples below. However, the same techniques are applicable to your data.
+
+.. image:: /images/tutorials/clickhouse-metabase/create.gif
+   :alt: Animated GIF showing creation of a service
+
+Setting up OSS version of Metabase
+------------------------------------------
+
+Depending on your operating system and preferences, you can choose among two approaches when `setting up the open source edition of Metabase <https://www.metabase.com/start/oss/>`_: using a Docker image and running Metabase as a jar.
+
+In this tutorial we'll be using `Docker <https://www.docker.com/>`_. However, similar results can be achieved using the ``.jar`` file.
+
+If you don't have a license token for a paid version of Metabase and you don't need a production installation, these steps `for the quick start <https://www.metabase.com/docs/latest/installation-and-operation/running-metabase-on-docker#open-source-quick-start>`_ plus a minor adjustment `to include a ClickHouse driver <https://www.metabase.com/docs/latest/installation-and-operation/running-metabase-on-docker#adding-external-dependencies-or-plugins>`_ is what you need.
+
+When starting your Docker container, make sure that you have access to the mounted folder. We'll need this folder for the next step.
+
+.. code:: bash
+
+    docker run -d -p 3000:3000 --name metabase metabase/metabase
+
+.. code:: bash
+
+    docker run -d -p 3000:3000 \
+    --mount type=bind,source=/path-to-plugins-folder,destination=/plugins \
+    --name metabase metabase/metabase
```

Comment:
Do we only need the second of these examples?
And maybe explain briefly that `/path-to-plugins-folder` is the path you want *outside* docker?